docs/pages/zero-trust-access/zero-trust-access.mdx

```diff
@@ -1,6 +1,289 @@
+          icon: "github",
+        },
+        {
+          name: "Entra ID",
+          href: "./sso/entra-id-oidc/",
+          icon: "entraId",
+        },
+       {
+          name: "More IdPs",
+          href: "./sso/",
+          arrow: true,
+        },
+      ]
+    },
+     {
+      title: "VNet: Build without VPNs",
+      description: "Connect to internal, non-browser TCP and SSH resources without VPNs. Use familiar tools and workflows while eliminating exposure to public internet.",
+      href: "../enroll-resources/application-access/guides/vnet/",
+      tags: [
+        {
+          name: "Teleport VNet Demo",
+          href: "https://youtu.be/OQ98VErAorM",
+          icon: "youtube",
+        },
+        {
+          name: "VNet Guide",
+          href: "../connect-your-client/vnet/",
+        },
+      ]
+    },
+    {
+      title: "Manage and Track Teleport Audit Events",
+      description: "Log granular audit events when users and services interact with your cluster. View audit events in Teleport or export them to a 3rd-party tool.",
+      href: "./export-audit-events/",
+      tags: [
+        {
+          name: "Elastic Stack",
+          href: "./export-audit-events/elastic-stack/",
+          icon: "elasticStack",
+        },
+        {
+          name: "Panther",
+          href: "./export-audit-events/panther/",
+          icon: "panther",
+        },
+        {
+          name: "Datadog",
+          href: "./export-audit-events/datadog/",
+          icon: "datadog",
+        },
+        {
+          name: "Splunk",
+          href: "./export-audit-events/splunk/",
+        },
+      ]
+    },
+    {
+      title: "Self-Host Teleport",
+      description: "Run Teleport Enterprise in your own infrastructure, with guides covering high availability and multi-region clusters, securing key material with KMS or HSMs, and more.",
+      href: "./deploy-a-cluster/",
+      tags: [
+        {
+          name: "AWS KMS",
+          href: "./deploy-a-cluster/aws-kms/",
+          icon: "aws",
+        },
+        {
+          name: "Google Cloud KMS",
+          href: "./deploy-a-cluster/gcp-kms/",
+          icon: "googleCloud",
+        },
+        {
+          name: "Helm Chart",
+          href: "./deploy-a-cluster/helm-deployments/",
+        },
+      ]
+    },  
+    {
+      title: "Infrastructure as Code",
+      description: "Manage Teleport's Dynamic Resources using infrastructure as code tools, including Terraform, Helm and the Teleport tctl client tool.",
+      href: "./infrastructure-as-code/",
+      tags: [
+        {
+          name: "Terraform",
+          href: "./infrastructure-as-code/terraform-provider/",
+        },
+        {
+          name: "Kubernetes Operator",
+          href: "./infrastructure-as-code/teleport-operator/",
+        },
+        {
+          name: "Helm Chart",
+          href: "./infrastructure-as-code/teleport-operator/teleport-operator-helm/",
+        },
```

Comment:
I'd consider omitting this. This is the docs for deploying the operator with a helm chart, not for managing dynamic resources using IaC tools.